app/components/course-admin/submissions-page.js

```diff
@@ -15,30 +15,62 @@ export default class AdminCourseSubmissionsPageComponent extends Component {
     this.selectedSubmission = this.args.submissions.sortBy('createdAt').at(-1);
   }
 
+  get currentCourseStage() {
+    return this.args.filteredCourseStages[0];
+  }
   get currentLanguage() {
```

Comment:
@ryan-gang no empty line between function blocks - again inconsistent. Can you create a linear task to see if there's a lint rule available for this that we can enable? Surprised that prettier doesn't include this by default